Fatras/include/ActsFatras/Physics/EnergyLoss/BetheHeitler.hpp

```diff
@@ -9,11 +9,12 @@
 #pragma once
 
 #include "Acts/Material/MaterialSlab.hpp"
+#include "Acts/Utilities/UnitVectors.hpp"
 #include "ActsFatras/EventData/Particle.hpp"
 
 #include <array>
+#include <iostream>
 #include <random>
-
```

Comment:
The `iostream` include is unused and can be remove. The empty line should stay.

Reply:
Done